test/integration/graql/query/GraqlDefineIT.java

```diff
@@ -407,104 +143,14 @@ public void whenDefiningAThing_Throw() {
         tx.execute(Graql.define(var("x").isa("movie")));
     }
 
-    @Test
-    public void whenModifyingAThingInADefineQuery_Throw() {
-        ConceptId id = tx.getEntityType("movie").instances().iterator().next().id();
-
-        exception.expect(GraqlSemanticException.class);
-        exception.expectMessage(anyOf(
-                is(GraqlSemanticException.defineUnsupportedProperty(Graql.Token.Property.HAS.toString()).getMessage()),
-                is(GraqlSemanticException.defineUnsupportedProperty(Graql.Token.Property.VALUE.toString()).getMessage())
-        ));
-
-        tx.execute(Graql.define(var().id(id.getValue()).has("title", "Bob")));
-    }
-
-    @Test @Ignore
-    public void whenSpecifyingLabelOfAnExistingConcept_LabelIsChanged() {
```

Comment:
i think we should open a new issue in the `graql` repository to bring this feature back: ability to change the label of a type using graql itself. You can document that the old way to do this was by awkwardly combining an ID with type(), which look like two setters or two getters and not a mixture. We need to rethink how this should be properly designed again in the future, for now it is banned via graql and only allowed via concept API

Reply:
Sounds good

https://github.com/graknlabs/graql/issues/131